### DISCUSSION TITLE
How to enable diagnostic messages?

### DISCUSSION THREAD

**Discussion by mrjones2014** [nvim-neotest/neotest#92] [Q&A] [answered]
The help under `:h neotest.diagnostic` isn't very clear. `vim.diagnostic.config()` expects a number for the namespace parameter, but the help text for neotest says to use the string `'neotest'`.

**Comment by rcarriga** [comment, marked as answer]
You can get the namespace number using `nvim_create_namespace("neotest")` and provide that to `vim.diagnostic.config()`

**Reply by mrjones2014** [reply]
ah okay perfect, thanks.
